tuna/session.py

```diff
@@ -67,14 +67,30 @@ class Session(BASE):
 
   def add_new_session(self, args, worker):
     """Add new session entry"""
-    self.arch = worker.machine.arch
-    self.num_cu = worker.machine.num_cu
-    self.rocm_v = worker.get_rocm_v()
-    self.miopen_v = worker.get_miopen_v()
     self.reason = args.label
-    if args.ticket:
-      self.ticket = args.ticket
     self.docker = args.docker_name
+    if hasattr(args, 'arch') and args.arch:
```

Comment:
why do we need to check hasattr & args.<>? is

Reply:
In case args is passed to the function without those attributes.

Reply:
Wouldnt pylint complain about that ?